contrib/babelfishpg_tsql/sql/sys_procedures.sql

```diff
@@ -271,3 +271,227 @@ GRANT EXECUTE on PROCEDURE sys.sp_babelfish_volatility(IN sys.varchar, IN sys.va
 
 CREATE OR REPLACE PROCEDURE sys.bbf_set_context_info(IN context_info sys.VARBINARY(128))
 AS 'babelfishpg_tsql' LANGUAGE C;
+
+CREATE OR REPLACE PROCEDURE sys.babelfish_exec_extendedproperty
+(
+  procedure_name text,
+  "@name" sys.sysname,
+  "@value" sys.sql_variant,
+  "@level0type" VARCHAR(128) = NULL,
+  "@level0name" sys.sysname = NULL,
+  "@level1type" VARCHAR(128) = NULL,
+  "@level1name" sys.sysname = NULL,
+  "@level2type" VARCHAR(128) = NULL,
+  "@level2name" sys.sysname = NULL
+)
+AS $$
+DECLARE
+  var_object_id INT;
+  var_user TEXT;
+  var_dbid SMALLINT;
+  var_schema_name NAME;
+  var_major_name NAME;
+  var_minor_name NAME;
+  var_type TEXT;
+  var_object_name TEXT;
+  var_object_type TEXT[];
+BEGIN
+  "@level0type" := UPPER(RTRIM("@level0type"));
```

Comment:
I have concern about the performance of this PLPGSQL implementation. 
Could you please try how much time dose it take this stored procedure to create 10K extended properties?

Reply:
I called sp_addextendedproperty xx, xx, 'schema', 'dbo', 'table', 'test', 'column', 'id' 10K times and get 122 seconds.

![image](https://github.com/babelfish-for-postgresql/babelfish_extensions/assets/5339358/5549a45c-579d-44c7-9228-366983f70f7a)

Reply:
Thanks! 12ms per call would be enough for the moment